template/web/wp-content/themes/<%= slug %>/app/Repositories/Repository.php

```diff
@@ -1,28 +1,26 @@
 <?php
 /**
- * Parent repository class. Provides a very basic, fluent interface for interacting
- * with PostCollection/PostQuery classes.
+ * Parent repository class. Provides a very basic, fluent interface to interact
+ * with query
  *
  * @package Studiometa
  */
 
 namespace Studiometa\Repositories;
 
-use Timber\PostCollection;
-
 /** Class */
 abstract class Repository {
```

Comment:
**question:** Should we add an `Abstract` prefix to the class name the same way we add the `Interface` suffix?

We would then have an `AbstractRepository.php` file containing an `AbstractRepository` class.